xml/System.Runtime.Versioning/MinimumOSPlatformAttribute.xml

```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Base type for all platform-specific attributes. Primarily used to allow grouping in documentation.</summary>
```

Comment:
```suggestion
    <summary>Indicates that an API is supported for a specified platform or operating system. If a version is specified, the API cannot be called from an earlier version.</summary>
```

Reply:
Oh, i thought this is the definition for `OSPlatformAttribute` base type 😛